pyaerocom/io/read_ghost.py

```diff
@@ -334,6 +337,14 @@ def _eval_flags(self, vardata, invalidate_flags, ds):
         invalid = ~valid
         return invalid
 
+    @property
+    def META_KEYS(self):
+        if 'GHOST.EBAS' in self.data_id:
+            META_KEYS = self.EBAS_META_KEYS
+        elif 'GHOST.EEA' in self.data_id:
+            META_KEYS = self.EEA_META_KEYS
+        return META_KEYS
```

Comment:
```suggestion
        raise KeyError('No metadata keys available for {}'.format(self.data_id)) 
```

Reply:
Wouldn't this ensure that the KeyError is always raised?
```suggestion
        else:
            raise KeyError('No metadata keys available for {}'.format(self.data_id))
        return META_KEYS
```